### ISSUE TITLE
bh.rotate() doesn't seem to delete anything on s3

### ISSUE THREAD

**Issue by stephantual** [tsileo/bakthat#27]
Hi, got a little issue with rotate():

```
with BakHelper("mongobak",
           destination=DESTINATION,
           password=PASSWORD,
           tags=["mongodb"]) as bh:
logging.info(sh.mongodump(**MONGODUMP_PARAMS))

bh.backup()
bh.rotate()
```

The backup work, upload to s3 encrypted and tarred as expected, however rotate() never rotates them. As per the conf file below I was expecting 2 full days of backup and one copy kept a month.

Conf file info:

```
  [xxxx@xxx xxx_scripts]$ sudo cat /xxxxxx/.bakthat.yml
  default:
    access_key: xxxxxxxxxxx
    default_destination: s3
    glacier_vault: xxxxxxxx-ice
    region_name: us-east-1
    rotation:
      days: 2
      first_week_day: 0
      months: 12
      weeks: 4
    s3_bucket: xxxx-backups-xxx
    secret_key: xxxxxxxxxxxxxxxxxxxxxx
```

Note: my backup script runs several backups a day, ie once an hour.

**Comment by tsileo** [comment]
Thanks for your feedback, I think I solved the issue, can you please try to install from git (4c7b8c2a2b58b9b225e6e84eecc62211b477a415) and tell me if it fix the bug for you ?

Keep me updated!

**Comment by tsileo** [comment]
There was an issue with bakthat 0.5.2, I just released bakthat 0.5.3, now it shoud works, keep me updated !

**Comment by stephantual** [comment]
This is now corrected indeed, thank you! Sorry it took me a while to reply, had to fight pip :)
